api/proto/config/service_enabled.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+package proto.config;
+
+option go_package = "github.com/spinnaker/kleat/api/client/config";
+
+// Configuration for whether a particular microservice is enabled.
+message ServiceEnabled {
```

Comment:
Do we anticipate that this could ever have other information?  Or maybe you deliberately chose to include `Enabled` in the name to make it clear that this is not a place for service-discovery type information?

Reply:
Yeah, I wish there was a way to lock this proto so no other fields end up in here...definitely should only include `enabled`; if we feel like other stuff belongs in here I think we need to re-evaluate the entire approach of this even being part of individual service settings.

Reply:
Yup, makes sense! Probably overkill to add `reserved 2 to 536,870,911` to the message 😁 .